protected/humhub/modules/space/models/Space.php

```diff
@@ -217,6 +219,16 @@ public function afterSave($insert, $changedAttributes)
             $activity->source = $this;
             $activity->originator = $user;
             $activity->create();
+
+            // Auto enable xcoin module if installed & create DEFAULT & ISSUE Accounts
```

Comment:
same for here add an event and handle it in xcoin module

Reply:
> Why not trigger an event (or use predefined one) here and handle it elsewhere, we must not interfere a module logic with the core

-> maybe here you're last comment has meaning , because we can listen on `EVENT_AFTER_INSERT` for active records , so that when creating a space then we can enable automatically ethereum part in the xcoin module , but since i've already made changes on the core , i don't see the utility here.